### DISCUSSION TITLE
[SOLVED] Reupload files deleted remotely

### DISCUSSION THREAD

**Discussion by epsilveira** [abraunegg/onedrive#1357] [Q&A] [answered]
Hi, don't know if my configuration is wrong or this is expected.
I deleted some files on OneDrive (in other computer, but sync with online). After, my onedrive (Linux) didn't delete the files (as I expect), but uploaded again the files there wasn't on remote/online OneDrive.
Is there an option on configuration to synchronize the changes?

Example in journalctl log:
`mar 19 20:23:03 Eddie onedrive[4491]: Uploading new file name_file.txt ... done.`

My config is with all default, except the lines:
```
skip_dir = "some_folders"
skip_file = "~*|.~*|*.tmp"
rate_limit = "131072"

```
onedrive is running with --monitor:
```
# systemctl --user status onedrive
● onedrive.service - OneDrive Free Client
     Loaded: loaded (/usr/lib/systemd/user/onedrive.service; enabled; vendor preset: enabled)
     Active: active (running) since Fri 2021-03-19 20:04:46 -03; 45min ago
       Docs: https://github.com/abraunegg/onedrive
   Main PID: 4491 (onedrive)
     CGroup: /user.slice/user-1000.slice/user@1000.service/onedrive.service
             └─4491 /usr/bin/onedrive --monitor

```

Thank you!

**Comment by abraunegg** [comment, marked as answer]
@epsilveira 
There could be any number of reasons here:

1. A race condition between your systems
2. Configuration issue
3. Version issue

Remote changes are downloaded regardless of method of operation. By default, the onedrive client on Linux syncs remote changes (add, rename, deletes etc) every 5 mins and this is governed by the 'monitor_interval' option.

Potentially, you did not wait for the sync process to occur.

**Reply by epsilveira** [reply]
Thank you for the (fast) answer! 

I will made some tests, but if the client syncs deletes, should be some of the issues mentioned.

Thank you again.

**Reply by epsilveira** [reply]
Just made some tests. Deleted files and wait. It's working perfectly. 

Thank you, I started this week with the Linux client.
